### PR TITLE
e2e update: updated integration Test and Enterprise Contracts Test

### PR DESCRIPTION
**Pull request by subhashkhileri**
## Fixes 

https://issues.redhat.com/browse/HAC-4267

## Description

e2e update: updated integration Test and Enterprise Contracts Test
Update ImageBundle to github repo for following scenarios :

1. Integration test pass pipeline
2. Integration test fail pipeline
3. Enterprise Contract pipeline and validation.
4. Integration test add and update.

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): Test

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: integration-tests/tests/advanced-happy-path.spec.ts [hunk 1]
```diff
@@ -378,18 +376,18 @@ describe('Advanced Happy path', () => {
 
     it('Verify EC Integration Test pipeline runs Logs Tab', () => {
       UIhelper.clickTab('Logs');
-      DetailsTab.verifyLogs('"result": "FAILURE"');
+      DetailsTab.verifyLogs('"result": "SUCCESS"');
```
Comment (Katka92) [inline integration-tests/tests/advanced-happy-path.spec.ts:379]: I guess now we are not checking a Failure anywhere, right? And I assume we are OK with that, right? :smiley:

Comment (subhashkhileri) [reply integration-tests/tests/advanced-happy-path.spec.ts:379]: earlier the Enterprise contract image bundle used to report failure one rule.
now we are using the git repo and in that all the rules reports Success.

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Comment by subhashkhileri** [comment]
[test]

**Pull request merged**